video_xblock/mixins.py

```diff
@@ -530,22 +530,37 @@ class SettingsMixin(XBlock):
 
     Sample default settings in /edx/app/edxapp/cms.env.json:
     "XBLOCK_SETTINGS": {
-      "video_xblock": {
+      "domain.com": {
```

Comment:
Here and everywhere, use `example.com` instead of `example.com`.

[Example.com is established to be used for illustrative examples in documents](http://example.com/).